backend/sirius-web-diagrams/src/main/java/org/eclipse/sirius/web/diagrams/ViewCreationRequest.java

```diff
@@ -107,7 +107,7 @@ public Builder targetObjectId(String targetObjectId) {
 
         public ViewCreationRequest build() {
             ViewCreationRequest viewCreationRequest = new ViewCreationRequest();
-            viewCreationRequest.parentElementId = Objects.requireNonNull(this.parentElementId);
+            viewCreationRequest.parentElementId = this.parentElementId;
```

Comment:
Why does that have to change in this PR? On top of that, we already have view creation request support where the parent is the diagram and the value is not null. See `CreateViewOperationHandler#handle` for example. `null` is not an accepted value here on purpose, don't change that.

Reply:
Done